WebUI/electron/subprocesses/aiBackendService.ts

```diff
@@ -52,17 +56,33 @@ export class AiBackendService extends LongLivedPythonApiService {
         status: 'executing',
         debugMessage: `installing dependencies`,
       }
+      await installHijacks()
+      const archToRequirements = (deviceArch: Arch) => {
+        switch (deviceArch) {
+          case 'arl_h':
+            return 'arl_h'
+          case 'acm':
+          case 'bmg':
+          case 'lnl':
+          case 'mtl':
+            return 'xpu'
+          default:
+            return 'unknown'
+        }
+      }
```

Comment:
Can we throw a more straightforward error when device arch is not supported, instead of reporting "requirements-unknown.txt" is not found?
There're complains about the "requirements-unknown.txt" like: https://github.com/intel/AI-Playground/issues/226